features/utilities/src/main/feature/feature.xml

```diff
@@ -55,13 +55,19 @@
         <bundle>mvn:ddf.action.core/action-core-impl/${project.version}</bundle>
     </feature>
 
-    <feature name="error" version="${project.version}"
-             description="Platform error handler.">
+    <feature name="error-api" version="${project.version}"
+             description="Platform error handler API.">
         <bundle>mvn:ddf.platform.error/platform-error-api/${project.version}</bundle>
-        <bundle>mvn:ddf.platform.error/platform-error-impl/${project.version}</bundle>
+        <bundle>mvn:ddf.platform.error/platform-error-servlet/${project.version}</bundle>
```

Comment:
❗️ The `platform-error-servlet` will need to be added to the pom of this module. Make sure to build this module to be sure that it's all 👍

Reply:
👍